plugins/quick-brick-google-admob-inline-banner/src/BannerWrapper.js

```diff
@@ -23,7 +24,7 @@ export default function BannerWrapper(props) {
   }
 
   return (
-    <View style={[styles.container, { backgroundColor: bannerBackground }]}>
+    <View style={[styles.container, { backgroundColor: bannerBackground }, accessibilityLabel]}>
```

Comment:
Seems that `accessibilityLabel` should be put as a prop to the View, but here it is put as a style prop.

Reply:
@oduk-sc fixed.